frontend/src/components/SQRDataTable.vue

```diff
@@ -28,13 +39,15 @@
 <script lang="ts">
 import { Component, Vue, Prop } from "vue-property-decorator";
 import _ from "lodash";
+import SQRTableData from "@/components/SQRTableData";
```

Comment:
As soon as this component is not SQR-specific, consider naming it in a more generic way.